test/unit/lodash.spec.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var Guy = require('./Guy');
+var chance = require('chance').Chance();
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function () {
+        describe('clamp', function () {
```

Comment:
It looks better when whole code is wrote in same way, so if position of curly braces is same throughout all project. Here you have one place where curly brace is in new line when in each other place it is in the same line.